packages/lib/src/language/locales/en-US.json

```diff
@@ -68,6 +68,7 @@
     "dateOfBirth": "Date of birth",
     "shopperEmail": "Email address",
     "gender": "Gender",
+    "gender.notselected": "Select a gender",
```

Comment:
Did you request translations for it?

Reply:
They still need to be done. But the english translation will work for now re. the a11y re-test